### DISCUSSION TITLE
kitty.conf(5) manual page structure

### DISCUSSION THREAD

**Discussion by grahamperrin** [kovidgoyal/kitty#6553] [General]
At a glance, <https://man.freebsd.org/cgi/man.cgi?query=kitty.conf&sektion=5&manpath=freebsd-ports> and <https://manpages.debian.org/unstable/kitty/kitty.conf.5.en.html> appear to be representations of: 

- <https://sw.kovidgoyal.net/kitty/conf/>. 

Please, is there any way for these representations to be more like the structure that's normally associated with man pages? 

Typically, the _NAME_ section of a man page includes a one-line description. The multi-line effect is noticeable at, for example, the tail of <https://man.freebsd.org/cgi/man.cgi?query=glob&apropos=1>: 

![image](https://github.com/kovidgoyal/kitty/assets/192271/927f7a50-a95a-4fb0-9092-d39d0e52f1b1)

Related: <https://www.freshports.org/x11/kitty/> for the port to FreeBSD; @nivit

**Comment by kovidgoyal** [comment]
https://github.com/kovidgoyal/kitty/commit/f299549dba3e4ae0232e6df5fa74126c9f20b11a

**Reply by grahamperrin** [reply]
Thank you!
